llama-index-integrations/llms/llama-index-llms-reka/pyproject.toml

```diff
@@ -0,0 +1,57 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+# [tool.llamahub.class_authors]
+CLASS = "rekaai"
+check-filenames = true
+check-hidden = true
+# [tool.llamahub]
+# contains_example = true
+import_path = "llama_index.llms.reka"
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
```

Comment:
lets add the tool.llamahub and tool.llamahub.class_authors sections (see other integrations for what this looks like)